addons/website_event/views/event_templates_page_misc.xml

```diff
@@ -11,20 +11,22 @@
 <!-- Multipage event - Default template for the Introduction page -->
 <template id="template_intro">
     <t t-call="website_event.layout">
-        <div class="oe_structure oe_empty" id="oe_structure_website_event_intro_1"/>
+        <t t-set="editor_message">DROP BUILDING BLOCKS HERE TO MAKE THEM AVAILABLE ACROSS ALL EVENTS INTRODUCTION PAGE</t>
+        <div class="oe_structure oe_empty" id="oe_structure_website_event_intro_1" t-att-data-editor-message="editor_message"/>
```

Comment:
I have this
![image](https://github.com/odoo/odoo/assets/78849981/1411b4d3-11f7-47b9-b7cd-8af9cf355e66)
If I add a block where the red arrow is, it's only on the current event intro

Reply:
Nice catch, this template is actually copied by record so it's not really shared.